linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml

```diff
@@ -52,4 +52,4 @@ warnings:
         See <b>{{{ weblink(link="http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/1401vend.htm") }}}</b>
         for a list of FIPS certified vendors.
 
-platform: machine
```

Comment:
@shawndwells is the intention of this change to substitute the way we detect a machine environment?
Currently we say that if it is not a container, it is a machine.

If a machine compiles its own kernel, would there be a kernel package?

Reply:
@shawndwells I suggest to stick to current detection of `machine` for now, and move and discuss addition of `kernel` platform to another PR.